build.gradle

```diff
@@ -19,10 +19,11 @@ buildscript {
 	repositories {
 		mavenCentral()
 		gradlePluginPortal()
+		maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
```

Comment:
Can be removed

Reply:
Ah we still need it for lwjgl3 release with that support. So probably can't be merged until then. Very cool though

Reply:
lwjgl3 is released🎉